docs/getting-data/index.md

```diff
@@ -4,7 +4,7 @@ should use for which use case.
 ## Current state & actions
 
 | Operation                                     | `@twurple/api`           | `@twurple/chat`                                        |
-|-----------------------------------------------|--------------------------|--------------------------------------------------------|
+| --------------------------------------------- | ------------------------ | ------------------------------------------------------ |
```

Comment:
Please don't change the formatting of the tables

Reply:
I didn't. VSCode must have done this on save/commit. Regardless, fixed.